File: Java/Car.java
class Car {
    Integer id;
    String license;
    Account driver;
    Integer passenger;

    public Car(String license, Account driver){ // Parametros obligatorios para poderse crear un objeto Car
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        System.out.println("License: " + license + "    Driver: " + driver.name);
    }
}

File: JavaScript/Car.js
class Car {    
    constructor(license, driver){
        this.license = license;
        this.driver = driver;
        this.passenger;
        this.id;
    }
    
    printDataCar() {
        console.log(this.driver);
        console.log(this.license);
        console.log(this.driver.name);
        console.log(this.driver.document);
        console.table(this.driver); // lo imprime como una tabla
    }
}

